server/index.js

```diff
@@ -0,0 +1,15 @@
+/*
+
+*/
+
+
+var express= require("express");
+var path = require("path");
+var bodyParser = require("body-parser");
+
+var app = express();
+var PORT = "3010";
+app.use(express.static(path.resolve(__dirname, '../client/')));
```

Comment:
to handle requests to `http://localhost:3000/<specific listing ID>` (to render modules dynamically depending on a listing) this statement should have a first argument e.g. `app.use('/:listingid', express.static(path.resolve(__dirname, '../client/')));`